app/code/Magento/Bundle/Block/Adminhtml/Catalog/Product/Edit/Tab/Bundle/Option/Search/Grid.php

```diff
@@ -190,18 +192,22 @@ public function getGridUrl()
     }
 
     /**
+     * Get selected products
+     *
      * @return mixed
      */
     protected function _getSelectedProducts()
     {
         $products = $this->getRequest()->getPost(
             'selected_products',
-            explode(',', $this->getRequest()->getParam('productss'))
+            explode(',', $this->getRequest()->getParam('productss') ?? '')
```

Comment:
Are you sure it shouldn't be like this?

```suggestion
            explode(',', $this->getRequest()->getParam('products') ?? '')
```

Reply:
@ihor-sviziev Hi! I think it's ok. Please check the method that create an Url

https://github.com/magento/magento2/blob/0af0fba365a4268c3f2d5c7168fcf17dbeab148c/app/code/Magento/Bundle/Block/Adminhtml/Catalog/Product/Edit/Tab/Bundle/Option/Search/Grid.php#L188

Reply:
Feels like that's the bug as well.
But ok, if we have a typo in both places - let's leave it as is for now